cclib/parser/psi4parser.py

```diff
@@ -1157,19 +1165,35 @@ def parse_gradient(self, inputfile, skip_lines):
     @staticmethod
     def parse_vibration(n, inputfile):
 
-        #   Freq [cm^-1]                1501.9533           1501.9533           1501.9533
-        #   Irrep
-        #   Reduced mass [u]              1.1820              1.1820              1.1820
-        #   Force const [mDyne/A]         1.5710              1.5710              1.5710
-        #   Turning point v=0 [a0]        0.2604              0.2604              0.2604
-        #   RMS dev v=0 [a0 u^1/2]        0.2002              0.2002              0.2002
-        #   Char temp [K]               2160.9731           2160.9731           2160.9731
-        #   ----------------------------------------------------------------------------------
-        #       1   C               -0.00  0.01  0.13   -0.00 -0.13  0.01   -0.13  0.00 -0.00
-        #       2   H                0.33 -0.03 -0.38    0.02  0.60 -0.02    0.14 -0.01 -0.32
-        #       3   H               -0.32 -0.03 -0.37   -0.01  0.60 -0.01    0.15 -0.01  0.33
-        #       4   H                0.02  0.32 -0.36    0.01  0.16 -0.34    0.60 -0.01  0.01
-        #       5   H                0.02 -0.33 -0.39    0.01  0.13  0.31    0.60  0.01  0.01
+        #  Vibration                       1                   8                   9
+        #  Freq [cm^-1]                698.2090i           1675.6707           1675.6899
+        #  Irrep                           B2                  B1                  A1
+        #  Reduced mass [u]              1.1842              1.0783              1.0783
+        #  Force const [mDyne/A]        -0.3401              1.7838              1.7839
+        #  Turning point v=0 [a0]        0.0000              0.2581              0.2581
+        #  RMS dev v=0 [a0 u^1/2]        0.0000              0.1895              0.1895
+        #  IR activ [km/mol]            695.8115            139.3986            139.4122

```

Comment:
You might want to say something here about IR intensity only being available for >= 1.7.